src/lib/input_schema.ts

```diff
@@ -102,3 +103,42 @@ export const createPrefilledInputFileFromInputSchema = async (actorFolderDir: st
         await writeJsonFile(inputJsonPath, inputFile);
     }
 };
+
+export const getDefaultsAndPrefillsFromInputSchema = (inputSchema: any) => {
```

Comment:
Comments please, especially for exported functions

Reply:
I'll add these in a follow up PR